### PR TITLE
Most used words search

### PR DESCRIPTION
**Pull request by Winwardo**

### CODE REVIEW

File: src/client/tweetAnalysis.js [hunk 1]
```diff
@@ -1,12 +1,54 @@
...
...esn't", 'doing', "don't", 'down', 'during', 'each', 'few', 'for', 'from', 'further', 'had', "hadn't", 'has', "hasn't", 'have', "haven't", 'having', 'he', "he'd", "he'll", "he's", 'her', 'here', "here's", 'hers', 'herself', 'him', 'himself', 'his', 'how', "how's", 'i', "i'd", "i'll", "i'm", "i've", 'if', 'in', 'into', 'is', "isn't", 'it', "it's", 'its', 'itself', "let's", 'me', 'more', 'most', "mustn't", 'my', 'myself', 'no', 'nor', 'not', 'of', 'off', 'on', 'once', 'only', 'or', 'other', 'ought', 'our', 'ours	ourselves', 'out', 'over', 'own', 'same', "shan't", 'she', "she'd", "she'll", "she's", 'should', "shouldn't", 'so', 'some', 'such', 'than', 'that', "that's", 'the', 'their', 'theirs', 'them', 'themselves', 'then', 'there', "there's", 'these', 'they', "they'd", "they'll", "they're", "they've", 'this', 'those', 'through', 'to', 'too', 'under', 'until', 'up', 'very', 'was', "wasn't", 'we', "we'd", "we'll", "we're", "we've", 'were', "weren't", 'what', "what's", 'when', "when's", 'where', "where's", 'which', 'while', 'who', "who's", 'whom', 'why', "why's", 'with', "won't", 'would', "wouldn't", 'you', "you'd", "you'll", "you're", "you've", 'your', 'yours', 'yourself', 'yourselves'];
```
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2194989253/issues/source?bid=3148782&fileBranchId=3167298#l4)

File: src/client/tweetAnalysis.js [hunk 2]
```diff
@@ -32,12 +74,17 @@ export const mostActiveUsers = (tweets) => {
...
   const wordCount = {};
   for (const tweet of tweets) {
-    for (const word of tweet.content.split(/[ !,.?"']/)) {
+    for (const word of tweet.content.split(/[ !,.?"'=]/)) {
       const trimmed = word.trim();
-      if (trimmed !== '') {
+      const lowerCase = trimmed.toLowerCase();
+      if (
+        (lowerCase !== '') &&
+        (stopList.indexOf(lowerCase) == -1) &&
```
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2194989253/issues/source?bid=3148782&fileBranchId=3167298#l85)

File: src/client/tweetAnalysis.js (hunk 1 above)
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Line is too long.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2194989253/issues/source?bid=3148782&fileBranchId=3167298#l4)

Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2195105034/issues/source?bid=3148782&fileBranchId=3167298#l4)

File: src/client/tweetAnalysis.js (hunk 2 above)
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2195105034/issues/source?bid=3148782&fileBranchId=3167298#l85)

File: src/client/tweetAnalysis.js (hunk 1 above)
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Codacy Issue found: [Line is too long.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2195105034/issues/source?bid=3148782&fileBranchId=3167298#l4)

**Comment by VasilyShelkov** [comment]
@Winwardo , this all looks good, especially  like how you've kept the search bar reusable by passing in the action name based on what it's searching :+1: 

only couple of questions before merging:
1. Is there a reason you're not using === instead of == ? 
2. how did you decide on what the stoplist is ?

**Comment by Winwardo** [comment]
1. Nope, Codacy caught that :)
2. I grabbed one from http://www.ranks.nl/stopwords, it's in tweetAnalysis.js with a comment. Let me know if this is okay, or if we should hold off for now with it / move it into its own file.

**Comment by Winwardo** [comment]
@VasilyShelkov ready for merge.

Referencing #5 
Fixes #50

File: src/client/tweetAnalysis.js (hunk 1 above)
Comment (Winwardo) [inline src/client/tweetAnalysis.js:4]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2223776510/issues/source?bid=3148782&fileBranchId=3167298#l4)

Comment (Winwardo) [inline src/client/tweetAnalysis.js:4]: Codacy Issue found: [Line is too long.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2223776510/issues/source?bid=3148782&fileBranchId=3167298#l4)

**Comment by VasilyShelkov** [comment]
close #50

**Pull request merged**